_config.yml

```diff
@@ -54,6 +54,12 @@ collections:
 sass:
   sass_dir: css
 
+# syntax highlighting
+markdown: kramdown
```

Comment:
Do I need to specify a version >= 2.3.1 here? Not sure if I'm reading this correctly:

![image](https://user-images.githubusercontent.com/25447342/113770628-91a03080-96df-11eb-989c-9f56cffd9cdf.png)

Reply:
Shouldn't need to go in our `_config`, I think the way to update kramdown would be to use `bundle update kramdown`. So long as that moves it above 2.3.1 in our Gemfile.lock, we should be OK

Reply:
Ok good to know, thanks! I will do that.